frontend/src/components/Modal/RiderModal.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState, Dispatch, SetStateAction } from 'react';
+import Modal from './Modal';
+import { Button } from '../FormElements/FormElements';
+import { ObjectType, Rider } from '../../types/index';
+import RiderModalInfo from './RiderModalInfo';
+import styles from './ridermodal.module.css';
+
+
+type RiderModalProps = {
+  riders: Array<Rider>;
+  setRiders: Dispatch<SetStateAction<Rider[]>>;
+}
+
+function addRider(newRider: Rider, allRiders: Rider[]) {
+  async function addBackend() {
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        firstName: newRider.firstName,
+        lastName: newRider.lastName,
+        phoneNumber: newRider.phoneNumber,
+        email: newRider.email,
+        accessibilityNeeds: newRider.accessibilityNeeds,
```

Comment:
The field "organization" is being added to the Rider schema, so this may need to be updated soon (but for now it looks good!)